info.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pyhardmode",
-  "version": "1.3.19",
+  "version": "1.3.20",
```

Comment:
Please no changelog or version updates in the HM repo. Both changelog and infojson are built by 
https://github.com/fgardt/factorio-mod-template